src/data_tools/src/jsf_data.cpp

```diff
@@ -363,6 +441,25 @@ std_data::sss_ping::PingsT convert_to_xtf_pings(const jsf_sss_ping::PingsT& ping
 }
 
 
+jsf_sss_ping::PingsT match_sound_vel(jsf_sss_ping::PingsT& sss_pings, jsf_dvl_ping::PingsT& dvl_pings)
+{
+    std::stable_sort(dvl_pings.begin(), dvl_pings.end(), [](const jsf_dvl_ping& dvl_ping1, const jsf_dvl_ping& dvl_ping2) {
```

Comment:
Ok, so my suspicion are actually these two `stable_sort` invocations. There seems to be some template instantiation problem here that causes a `static_assert`.